homeassistant/components/services.yaml

```diff
@@ -601,3 +601,17 @@ nest:
       trip_id:
         description: Optional identity of a trip. Using the same trip_ID will update the estimation.
         example: trip_back_home
+
+elkm1:
+  sensor_speak_word:
```

Comment:
Move these to a new file `services.yaml` in the elkm1 package. See other components that are packages for example.

Reply:
Will do. Don't understand why though. What is different about elkm1 compared to the other services in where they were put?

Reply:
We don't want to add more service descriptions to the base component package file.

The descriptions in the base component file are always loaded and cached the first time we need all the service descriptions of registered services, eg in the frontend, so the longer that file is the longer time it will take to do that, and the more information, potentially not needed info, will be stored in memory.

That's why we want to split this file up and put the descriptions in the other child components so that we only load the necessary information.

Reply:
Thanks, that helps. Sounds like that the services will be eventually moved. Makes a lot of sense. Would it make sense to leave a "trail" in that `services.yaml` that says what you said in your comment? It could help point people in the right direction.

Reply:
Make the suggestion in a PR. It should be documented in our dev docs at minimum.